code/modules/plumbing/plumbers/pill_press.dm

```diff
@@ -13,9 +13,9 @@
 	///pill name
 	var/pill_name = "factory pill"
 	///the icon_state number for the pill.
-	var/pill_shape = "pill_shape_capsule_purple_pink"
+	var/chosenPillStyle = "pill_shape_capsule_purple_pink"
 	///list of id's and icons for the pill selection of the ui
-	var/static/list/pill_styles = list()
+	var/static/list/pillStyles = list()
```

Comment:
Actively making the variable names worse :sob:

Reply:
Need some time to take care of this. Too busy.